Xamarin.Forms.Controls/GalleryPages/CollectionViewGalleries/CarouselViewGalleries/CarouselViewGallery.cs

```diff
@@ -34,5 +41,16 @@ public CarouselViewGallery()
 				}
 			};
 		}
+
+		private void Button_Clicked(object sender, System.EventArgs e)
```

Comment:
Can we drop private and call it ButtonClicked , thanks

Reply:
Done.